lib/validators/doc-media-validator.ts

```diff
@@ -1,9 +1,15 @@
 /**
  * Validates usage of doc-media directive
+ *
+ * Rules:
+ *  - A component is not allowed to have more than one doc-media directive.
+ *  - A component with 1 doc-media directives -can- have a property control type "media-properties"
```

Comment:
I think if a component has doc-media directive then it must have a property control type media-properties. Right, Sander?

Reply:
It's not required. It just means you don't have the dynamic social media properties in the sidebar.

Reply:
Ah right, sorry. It's the interactive directive requires the property.